docs/guides/deploy.md

```diff
@@ -75,6 +75,6 @@ If you want to have your applications sign blockchain transactions, see [Sign a
 To run the [Demo website](../../examples/#tonomy-demo-integration-application) in your network follow these extra steps:
```

Comment:
we also need to add instructions (before step 2) to deploy the oncoin.token contract